src/main/java/mops/businesslogic/GroupServiceImpl.java

```diff
@@ -14,14 +14,17 @@
      * Directory Service.
      */
     private DirectoryService directoryService;
+    /**
+     * API to GruppenFindung.
+     */
+    private PermissionService permissionService;
 
     /**
      * {@inheritDoc}
      */
     @Override
     public List<Group> getAllGroups(Account account) throws MopsException {
-        // TODO: get groups from Gruppenfindung
-        return List.of();
+        return permissionService.fetchGroupsForUser(account);
```

Comment:
use the rest template here, `fetchGroupsForUser` doesn't need to exist in `PermissionService`, as it has nothing to do with permissions.